src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -413,6 +413,8 @@ internal sealed partial class CultureData
         private static volatile Dictionary<string, CultureData>? s_cachedRegions;
         private static volatile Dictionary<string, string>? s_regionNames;
 
+        internal string? InteropName => _sWindowsName;
```

Comment:
Could you add a summary xml comment describing the purpose of this?  There are so many "XxName" properties in globalization, it's hard to keep track of what each means :)